anonymize.py

```diff
@@ -53,19 +54,26 @@ def get_updates(config):
                     updates.append("`%s` = INET_NTOA(RAND()*1000000000)" % field)
             elif operation == 'random_email':
                 for field in listify(details):
-                    updates.append("`%s` = CONCAT(id, '@mozilla.com')"
-                                   % field)
+                    updates.append("`%(field)s` = CONCAT(`%(n)s`, '@inbucket.plurall.net')" % dict(field=field,n=pkName))
```

Comment:
@williamvalentim maybe should you change this email "plurall.net" to something generic?

Reply:
I don't know. this domain like a mailinator for plurall.